versions/1/integration/data_stream/spec.yml

```diff
@@ -25,12 +25,7 @@ spec:
       name: agent
```

Comment:
What's the difference between the agent directory in a `data_stream` vs. one in the overall package? Can we document this in the spec?

Reply:
Agent in spec is used by few integrations, which don't need a data stream, but they are still integrations:

https://github.com/elastic/integrations/blob/main/packages/fleet_server/agent/input/agent.yml.hbs
https://github.com/elastic/package-storage/blob/production/packages/apm/8.2.0/agent/input/template.yml.hbs

I was confused too, but didn't want to merge "agent" definitions into single one. Let me know what do you think.